### ISSUE TITLE
Add example folder with common values.yaml scenarios

### ISSUE THREAD

**Issue by TylerHelmuth** [open-telemetry/opentelemetry-helm-charts#138]
It would be nice to have some example values.yaml for common scenarios customers might have.  Some scenarios could be:

- agent only configuration
- standalone only configuration
- agent and standalone
- Traces only
- Metrics only
- Logs only
- Common receivers (hostmetrics?)
- Common processors (resourceprocessor or k8sattributeprocessor?)
- Proxy configuration

Could be worth combining certain examples so we don't have too many values.yaml to maintain on breaking changes.

**Comment by TylerHelmuth** [comment]
I think this is blocked by #90.  I wanted to create an agent example with only the metrics pipeline but can't overwrite the default config so that trace and logs are excluded.

**Comment by TylerHelmuth** [comment]
Narrowing the scope of this to only 
- agent only configuration
- standalone only configuration
- agent and standalone
while we work towards #91
